mcp_server.py

```
@@ -591,6 +714,446 @@ def calculate_detailed_metrics(tree: ast.AST, source_code: str, include_halstead
     return result
 
 
+async def handle_update_documentation(arguments: Dict[str, Any]) -> List[TextContent]:
+    """Handle manual documentation update."""
+    global server_state
+    
+    force = arguments.get("force", False)
```

Ruff (0.11.9)

Local variable `force` is assigned to but never used

Remove assignment to unused variable `force`

(F841)